pkg/nfd-master/nfd-master.go

```diff
@@ -30,6 +28,7 @@ import (
 	"strings"
 	"time"
 
+	//"github.com/fsnotify/fsnotify"
```

Comment:
Remove?

Reply:
Woops 🙄  Thanks for pointing this out, fixed.